src/applications/vaos/routes.jsx

```diff
@@ -1,9 +1,17 @@
+import React from 'react';
+import { Route, IndexRoute } from 'react-router';
 import VAOSApp from './containers/VAOSApp';
 import Index from './containers/Main';
+import NewAppointmentLayout from './components/NewAppointmentLayout';
+import TypeOfAppointmentPage from './containers/TypeOfAppointmentPage';
 
-const routes = {
-  component: VAOSApp,
-  childRoutes: [{ path: '/', component: Index }],
-};
+const routes = (
+  <Route path="/" component={VAOSApp}>
```

Comment:
This format seems to work better when you have index routes.

Reply:
Did you use the yeoman generator to spin up the app initially?

Reply:
No, @narin didn't know about it and create the app from scratch. I did run it recently when I was doing some exploratory work and found a couple things that I need to put into tickets for you all, though.